Classes/Service/ContentEditableWrapperService.php

```diff
@@ -124,7 +124,7 @@ public function wrapContent(string $table, int $uid, array $dataArr, string $con
         // Could make it would make it possible to configure cid for use with extensions that create columns by content
         $class = 't3-frontend-editing__inline-actions';
         $content = sprintf(
-            '<%s class="t3-frontend-editing__ce %s" title="%s" data-movable="1"' .
+            '<%s class="t3-frontend-editing__ce %s" title="%s" draggable="true" data-movable="1"' .
```

Comment:
Be aware that this property do not behave the same in all browsers. Take an additional check in Firefox and Safari, it may in worst case make the element not editable.

Reply:
@MattiasNilsson is correct here. This appears to break or severely degrade editing in Safari (12.1) and Firefox (66.0.5).

@viacheslavresultify I suggest adding a draggable button to the inline editing actions.